hedera-node/src/test/java/com/hedera/services/store/contracts/precompile/HTSPrecompiledContractTest.java

```diff
@@ -516,6 +517,29 @@ void computeCallsCorrectImplementationForMintToken() {
 		assertTrue(subject.getPrecompile() instanceof HTSPrecompiledContract.MintPrecompile);
 	}
 
+	@Test
+	void computeCallsCorrectImplementationForMintToken2() {
```

Comment:
Can we give this test a more descriptive name? What is the difference between `computeCallsCorrectImplementationForMintToken` and `computeCallsCorrectImplementationForMintToken2`